server/test/getArtists.tests.js

```diff
@@ -0,0 +1,47 @@
+import { describe, it } from 'mocha';
+import chai from 'chai';
+import axios from 'axios';
+
+const expect = chai.expect;
+
+
+
+describe('getArtists', function () {
+  it('Should check getArtists endpoint and fetched data', async function () {
+
+    this.timeout(10000);
+
+    const requestBodies = [{"name": {"hebrew" : "אמן"}}, {"name": {"english" : "Eyal"}}, {"name": {"english" : "moshE"}}, {"name": {"arabic" : "فنان"}}]
```

Comment:
this tests assumes that there is such an object. you need to create this object as a part of "before" and delete it in "after"

Reply:
I updated the test accordingly, but I don't understand why to do that.